IISIntegration.sln

```diff
@@ -380,6 +382,18 @@ Global
 		{1533E271-F61B-441B-8B74-59FB61DF0552}.Release|x64.Build.0 = Release|x64
 		{1533E271-F61B-441B-8B74-59FB61DF0552}.Release|x86.ActiveCfg = Release|Win32
 		{1533E271-F61B-441B-8B74-59FB61DF0552}.Release|x86.Build.0 = Release|Win32
+		{1F0C8D9B-F47B-41F3-9FC9-6954B6DC7712}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Are P2P dependencies between test projects and native set correctly?

Reply:
They depend on .IIS, which depends on the native components.